### PR TITLE
[PRP-256] Tidy up e2e commands

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-256

- Replace overly long npm script for running e2e commands with standalone bash script
- Standardize app names in `participant/docker-compose.e2e.yml` and `staff/docker-compose.e2e.yml` so bash script is more uniform
- Remove default playwright pixel drift flexibility now that all snapshots are linux snapshots
- Update `<List>` component props

Follow on work from #34 

- In `participant` and in `staff`, run:
  - `npm run e2e`. Tests should run and pass
  - `npm run e2e:update`. Snapshots should update if necessary
  - `npm run e2e:cleanup`. Docker containers should be removed

### CODE REVIEW

**Comment by rocketnova** [comment]
Hmmm I wonder why these CI jobs failed 🤔 . Moving back to draft while I investigate.

File: participant/app/components/List.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import type { ReactElement } from "react";
+import type { i18nKey } from "app/types";
+import { Trans } from "react-i18next";
+
+export type ListProps = {
+  listKeys: i18nKey[];
+  ordered?: boolean;
```
Comment (microwavenby) [inline participant/app/components/List.tsx]: i kinda wish we were using an enum type here but 🤷🏻‍♀️
```suggestion
  type?: "ordered" || "unordered";
```

**Review by microwavenby — APPROVED** [review]

File: participant/app/components/List.tsx (hunk 1 above)
Comment (rocketnova) [reply participant/app/components/List.tsx]: Oh sure. I feel relatively neutral on this, so I'll change it.

File: participant/package.json [hunk 2]
```diff
@@ -16,9 +16,9 @@
     "lint": "eslint --cache --cache-location ./node_modules/.cache/eslint .",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "e2e": "docker compose -f docker-compose.e2e.yml up --build --wait database-e2e && docker compose -f docker-compose.e2e.yml up --build --wait remix-e2e && docker compose -f docker-compose.e2e.yml run --rm playwright npx playwright test --retries=3 --reporter=list",
-    "e2e:update": "docker compose -f docker-compose.e2e.yml up --build --wait database-e2e && docker compose -f docker-compose.e2e.yml up --build --wait remix-e2e && docker compose -f docker-compose.e2e.yml run --rm playwright npx playwright test --update-snapshots --reporter=list",
-    "e2e:cleanup": "docker compose -f docker-compose.e2e.yml down --remove-orphans",
+    "e2e": "../bin/run-e2e.sh test",
+    "e2e:update": "../bin/run-e2e.sh update",
+    "e2e:cleanup": "../bin/run-e2e.sh cleanup",
```
Comment (microwavenby) [inline participant/package.json:21]: 💜

Comment (microwavenby) [reply participant/package.json:21]: full circle 🔁

File: participant/stories/components/List.stories.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ export default {
 
 const defaultProps: ListProps = {
   listKeys: ["list-a", "list-b", "list-c"],
-  ordered: false,
+  type: "unordered",
```
Comment (microwavenby) [inline participant/stories/components/List.stories.tsx:11]: 💜

**Review by microwavenby — APPROVED** [review]

**Pull request merged**